js/controllers/search.js

```diff
@@ -67,13 +67,13 @@ SearchController.prototype.onChange_ = function() {
 };
 
 SearchController.prototype.onKeydown_ = function(e) {
-  switch (e.keyCode) {
-    case 13:
+  switch (e.key) {
+    case "Enter":
```

Comment:
Throughout this patch:

prefer `'` over `"` for JavaScript. prefer `"` when doing HTML.

Reply:
Done here and in other file, thanks for catching that I need to get better at these :\